DESCRIPTION

```diff
@@ -19,7 +19,8 @@ Authors@R: c(
     person("Pradeep", "Adhokshaja",  role = "ctb"),
     person("Olivia", "Barrows", role = "ctb"),
     person("Jojo", "Miller", role = "ctb"),
-    person("Jayla", "Nakayama", role = "ctb"))
+    person("Jayla", "Nakayama", role = "ctb"),
+    person("Natalia", "Iannucci", role = "ctb"))
```

Comment:
No prob if you don't want to include your email (preferably something that you'll even after you leave Smith). Just making sure that this was an explicit choice.